cpp/src/bitmask/bit_mask.cuh

```diff
@@ -17,221 +17,242 @@
 #ifndef CUDF_BIT_MASK_CUH
 #define CUDF_BIT_MASK_CUH
 
+#include <cuda_runtime.h>
 #include <utilities/cudf_utils.h>
+#include <types.hpp>
 #include <utilities/bit_util.cuh>
-#include <utilities/integer_utils.hpp>
 #include <utilities/error_utils.hpp>
+#include <utilities/integer_utils.hpp>
 
 namespace bit_mask {
-  typedef uint32_t     bit_mask_t;
-  enum { bits_per_element = gdf::util::size_in_bits<bit_mask_t>() };
-
-  /**
-   * @brief determine the number of bit_mask_t elements are used
-   *
-   * @param[in]  size    Number of bits in the bitmask
-   *
-   * @return the number of elements
-   */
-  CUDA_HOST_DEVICE_CALLABLE
-  constexpr gdf_size_type num_elements(gdf_size_type size) {
-//    return (( size + ( bits_per_element - 1)) / bits_per_element );
-    return cudf::util::div_rounding_up_unsafe<gdf_size_type>(size, bits_per_element);
-  }
+enum { bits_per_element = gdf::util::size_in_bits<bit_mask_t>() };
```

Comment:
The `enum`-trick is what people did for compile-time constants before `constexpr` was a thing.

There are some corner cases where the behavior is different between the two. See: https://stackoverflow.com/questions/37259807/static-constexpr-int-vs-old-fashioned-enum-when-and-why/37259949 

(TL;DR, `enum` is technically better because it's guaranteed to be a compile-time constant. `constexpr` _can be_ a compile time constant, and can lead to some link errors if odr-used.)

I see it mostly a style issue. Had I written this code, I would have used a `static constexpr`, but since I didn't, I just left it alone.